coops-ui/src/resources/config-objects.ts

```diff
@@ -0,0 +1,68 @@
+
```

Comment:
I like that you've created a new folder for these things but I think the filename should be more specific to the type of resource... otherwise what would you call another resource (won't it also be a config object?)? Maybe "business-config.ts"?